recipes/oslo.config/meta.yaml

```diff
@@ -42,7 +42,7 @@ test:
 about:
   home: http://docs.openstack.org/developer/oslo.config/
   license: Apache 2.0
-  # license_file: No MANIFEST.in
+  # license_file: No MANIFEST.in - see https://bugs.launchpad.net/oslo.config/+bug/1624979
```

Comment:
valid question - that is, I'm not sure what the explicit hard-link provides. That said, `conda build` gets angry if it can't find the file itself.